src/avatar.cpp

```diff
@@ -687,6 +691,26 @@ bool avatar::read( int inventory_position, const bool continuous )
             }
             act.index = menu.ret;
         }
+        if( it.type->use_methods.find( "MA_MANUAL" ) != it.type->use_methods.end() ) {
```

Comment:
```suggestion
        if( it.type->use_methods.count( "MA_MANUAL" ) ) {
```

Reply:
By the way, this method of identifying if a book is a martial arts manual seems realy weird to me. There has to be a better way, but out of scope of this PR, so I left it as is.

Reply:
Agreed, on both counts.